assets/docs/sources/tutorial/13_export_and_feature.md

```
@@ -0,0 +1,198 @@
+  ├── PluginFeature     ← 封装插件调用，参数根据来源自适应
+  └── 你的自定义 Feature  ← 继承 Feature，实现任意逻辑
+```
+
+- **Feature 基类**：通用的附加行为抽象，不绑定任何具体实现。默认在所有生命周期钩子中执行。
+- **PluginFeature**：Feature 的子类，专门封装 jmcomic 插件。除了调用插件之外，还会根据调用来源动态适配 `filename_rule`、`level` 等参数。
+
+### 执行流程
+
+Feature **自然嵌入到 downloader 的生命周期钩子**中自动触发：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)